cogs/sessions.py

```diff
@@ -59,7 +64,31 @@ async def autocomplete_credentials(self, interaction: Interaction, current: str)
             for credentials in await credentials_in_guild_tll(interaction.guild_id) if current.lower() in str(credentials).lower()]
         choices.append(app_commands.Choice(name="Custom", value=0))
         return choices
-    
+
+    async def autocomplete_end_time(self, _: Interaction, current: str):
+        if current != "":
+            return [
+                app_commands.Choice(name=current, value=current)
+            ]
+
+        choices = []
+        candidates = [
+            [1, 40],
```

Comment:
not sure about this one. I first added 1:30h, however, usually the capturing starts earlier then the game, and I don't want to miss events from the end, if the game is 90mins.

On the other hand, I usually used 2 hours+ only so far, and stopped the session manually. So it might also be worth adding this option at the beginning? Maybe you have some more insights on how people use the command right now (with regards to the end time).